nixos/modules/services/networking/gdomap.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gdomap;
+in
+{
+  #
+  # interface
+  #
+  options = {
+    services.gdomap = {
+      enable = mkOption {
```

Comment:
Note that `mkEnableOption` is a convenient short-hand for these types of option definitions.